modules/vpn/main.tf

```diff
@@ -96,7 +96,7 @@ resource "aws_s3_bucket" "furyagent" {
 }
 
 resource "aws_iam_user" "furyagent" {
-  name = "${var.name}-${var.vpc_id}-${data.aws_region.current.name}-vpn"
+  name = "${var.name}-${var.vpc_id}-${data.aws_region.current.name}-${local.vpn_bucket_iam_name_prefix}"
```

Comment:
I would leave vpn here, and deal with it in the migration procedure if necessary.